Resources/Audio/Effects/attributions.yml

```diff
@@ -93,6 +93,11 @@
   license: Custom
   source: https://gdc.sonniss.com
 
+- files: ["church_bell1.ogg", "church_bell2.ogg", "church_bell3.ogg", "church_bell4.ogg"]
+  copyright: '"Dramatic church bells" in "Church Bell" on mixkit.co. See https://mixkit.co/license/#sfxFree for license. Editted from stereo to mono, and variations pitched.'
+  license: Custom
+  source: https://mixkit.co/free-sound-effects/church-bell/
```

Comment:
> You can’t redistribute the Item on its own, as stock, in a tool or template, or with source files.

I'm seeking other opinions but this sounds like we can't include it?

Reply:
Sound changed